Submitty/Submitty
# [Feature:TAGrading] highlight pos/neg marks w/ count up/down

### What is the current behavior?
Closes #3413 

### What is the new behavior?
Now, when you change the count up/count down grading guideline, any component score that doesn't comply is highlighted. It becomes unhighlighted when you change it to comply.
### Other information?
Testing:
1. Sign in as instructor
2. Go to Gradeables
3. Click on the edit pencil
4. Click on rubric and select a component
5. Play around with possible scores and count up/down configurations
If the score is negative on count up, it should be highlighted and vice versa. Changing the score to comply should remove the highlight.
![Instructor highlight 2019-07-23 15-03](https://user-images.githubusercontent.com/36307579/61740062-be6db200-ad5b-11e9-8229-29dbbe5d9ae3.gif)
6. Sign in as TA and edit rhe grading rubric when you grade a student
![TAGrading highlight2019-07-23 11-46](https://user-images.githubusercontent.com/36307579/61740028-a7c75b00-ad5b-11e9-8962-ba00419fafa7.gif)
